lib/pliny/sidekiq/version.rb

```diff
@@ -1,5 +1,5 @@
 module Pliny
   module Sidekiq
-    VERSION = "0.3.2"
+    VERSION = "0.3.3"
```

Comment:
Changing the `sidekiq` version requirements like this warrants a bigger version bump IMO.

```suggestion
    VERSION = "0.4.0"
```